securitycenter/snippets_v2/snippets_mute_config_v2.py

```diff
@@ -0,0 +1,183 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config_v2]
+
+
+def create_mute_rule(parent_path: str, location_id: str, mute_config_id: str):
+    """
+     Creates a mute configuration under a given scope that will mute
+     all new findings that match a given filter.
+     Existing findings will NOT BE muted.
+     Args:
+         parent_path: use any one of the following options:
+                      - organizations/{organization_id}
+                      - folders/{folder_id}
+                      - projects/{project_id}
+         mute_config_id: Set a unique id; max of 63 chars.
+     """
+
+    from google.cloud import securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    mute_config = securitycenter_v2.MuteConfig()
+    mute_config.description = "Mute low-medium IAM grants excluding 'compute' "
+    # Set mute rule(s).
+    # To construct mute rules and for supported properties, see:
+    # https://cloud.google.com/security-command-center/docs/how-to-mute-findings#create_mute_rules
+    mute_config.filter = (
+        'severity="LOW" OR severity="MEDIUM" AND '
+        'category="Persistence: IAM Anomalous Grant" AND '
+        '-resource.type:"compute"'
+    )
+    mute_config.type = "STATIC"
+
+    request = securitycenter_v2.CreateMuteConfigRequest()
+    request.parent = parent_path+"/locations/"+location_id
+    request.mute_config_id = mute_config_id
+    request.mute_config = mute_config
+
+
+    mute_config = client.create_mute_config(request=request)
+    print(f"Mute rule created successfully: {mute_config.name}")
+    return mute_config
+
+# [END securitycenter_create_mute_config_v2]
+
+
+# [START securitycenter_delete_mute_config_v2]
+def delete_mute_rule(parent_path: str, location_id: str, mute_config_name: str) -> None:
+    """
+    Deletes a mute configuration given its resource name.
+    Note: Previously muted findings are not affected when a mute config is deleted.
+    Args:
+         parent_path: use any one of the following options:
+                     - organizations/{organization_id}
+                     - folders/{folder_id}
+                     - projects/{project_id}
+        mute_config_name: Set a unique id; max of 63 chars.
+    """
+    from google.cloud import securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.DeleteMuteConfigRequest()
+    request.name = parent_path+"/locations/"+location_id+"/muteConfigs/"+mute_config_name
+
+    client.delete_mute_config(request)
+    print(f"Mute rule deleted successfully: {mute_config_name}")
+
+
+# [END securitycenter_delete_mute_config_v2]
+
+
+# [START securitycenter_get_mute_config_v2]
+def get_mute_rule(parent_path: str, location_id: str, mute_config_name: str) -> None:
```

Comment:
nit: for uniformity please update `mute_config_name` to `mute_config_id`

Reply:
addressed